Ports/SuperTuxKart/patches/0003-Fixup-Vulkan-SDL-platform-quirks.patch

```diff
@@ -0,0 +1,210 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Dominika Liberda <ja@sdomi.pl>
+Date: Sun, 2 Jul 2023 05:49:46 +0200
+Subject: [PATCH] Fixup Vulkan/SDL platform quirks
+
+---
+ lib/graphics_engine/include/ge_main.hpp                  | 1 +
+ lib/graphics_engine/include/vk_mem_alloc.h               | 1 +
+ lib/graphics_engine/src/ge_main.cpp                      | 1 +
+ lib/graphics_engine/src/ge_spm_buffer.cpp                | 1 +
+ lib/graphics_engine/src/ge_vulkan_command_loader.cpp     | 1 +
+ lib/graphics_engine/src/ge_vulkan_depth_texture.cpp      | 1 +
+ lib/graphics_engine/src/ge_vulkan_draw_call.cpp          | 1 +
+ lib/graphics_engine/src/ge_vulkan_fbo_texture.cpp        | 1 +
+ lib/graphics_engine/src/ge_vulkan_fbo_texture.hpp        | 1 +
+ lib/graphics_engine/src/ge_vulkan_skybox_renderer.cpp    | 1 +
+ lib/graphics_engine/src/ge_vulkan_texture.cpp            | 1 +
+ lib/graphics_engine/src/ge_vulkan_texture_descriptor.cpp | 1 +
+ lib/irrlicht/source/Irrlicht/CIrrDeviceSDL.cpp           | 7 +------
+ src/graphics/irr_driver.cpp                              | 2 +-
+ 14 files changed, 14 insertions(+), 7 deletions(-)
+
+diff --git a/lib/graphics_engine/include/ge_main.hpp b/lib/graphics_engine/include/ge_main.hpp
+index 83ba0084f32a5593a96f8e8f2f6be7b7a8846dac..13ce04d0345bd56b66cc612071c0b152b37c0ca6 100644
+--- a/lib/graphics_engine/include/ge_main.hpp
++++ b/lib/graphics_engine/include/ge_main.hpp
+@@ -7,6 +7,7 @@
+ #include <cstdint>
+ #include <string>
+ #include <unordered_set>
++#include <stdexcept>
```

Comment:
is there a common header file we can slap stdexcept into rather than patching 15 files?

Reply:
we tried a few and nothing seemed to work. if memory serves correctly, some other cpp standard version does this by default.